sycl/test/lit.cfg.py

```diff
@@ -105,10 +106,16 @@ def getDeviceCount(device_type):
         if re.match(r".*cuda", result[1]):
             is_cuda = True;
 
+    # if we have found gpu and there is additional information, let's check
+    # whether this is CUDA device or not
+    if device_type == "gpu" and value > 0 and len(result[1]):
```

Comment:
Would it make sense to also move the CUDA check above into this condition?

Reply:
You are right, fixed.